README.md

```diff
@@ -114,6 +114,16 @@ To get started have a look at our [documentation on contributing](https://github
 
 Code released under the [MIT License](https://github.com/Girl-Code-It/Opportunity-Calendar-Frontend/blob/develop/LICENSE).
 
+## Opportunity-Calendar-Frontend is a part of these Open Source programs
```

Comment:
1. Shift this section above Table of contents
2. Change the heading to 'We're a part of these Open Source Programs 
3. Center align the heading

Reply:
@anirudhsai20 Any updates?

Reply:
I will update my PR within today.